lib/roger/partition.ex

```diff
@@ -186,6 +191,7 @@ defmodule Roger.Partition do
   defp stop_partition(id, state) do
     case Roger.GProc.whereis({:app_supervisor, id}) do
       pid when is_pid(pid) ->
+        :ok = Consumer.force_shutdown(id)
```

Comment:
Instead of stopping all these pids yourself, maybe introduce a dynamic-supervisor for a partition, and stop that one?